docs/services/data/capture-sync.md

```diff
@@ -295,12 +295,13 @@ The following attributes are available for data capture configuration:
 | ------------------ | ------ | --------- | ----------- |
 | `capture_frequency_hz` | float   | **Required** | Frequency in hertz at which to capture data. For example, to capture a reading every 2 seconds, enter `0.5`. |
 | `method` | string | **Required** | Depends on the type of component or service. See [Supported components and services](/services/data/capture-sync/#supported-components-and-services). |
+| `retention_policy` | object | Optional | Option to configure how long data collected by this component should remain stored in the Viam cloud. You must set this in JSON mode. See the JSON example for a camera component. Options: `"days": <int>`. |
```

Comment:
directing them to JSON example is a little awk but I feel like its necessary to point this out to make this usable. Also, with "options" there are going to be more than one key-value pair option at some point soon